sunpy/io/_file_tools.py

```diff
@@ -124,6 +149,8 @@ def read_file(filepath, filetype=None, **kwargs):
     pairs : `list`
         A list of (data, header) tuples.
     """
+    filepath = get_open_file(filepath, **kwargs)
+    filepath = filepath.path
```

Comment:
So what does this return?

Reply:
So this just bypasses the whole fsspec OpenFile by returning a filepath like before?